spring-rest2ts-generator/src/main/java/com/blueveery/springrest2ts/converters/ModelClassesToTsClassesConverter.java

```diff
@@ -0,0 +1,127 @@
+package com.blueveery.springrest2ts.converters;
+
+import com.blueveery.springrest2ts.implgens.ImplementationGenerator;
+import com.blueveery.springrest2ts.naming.ClassNameMapper;
+import com.blueveery.springrest2ts.tsmodel.TSClass;
+import com.blueveery.springrest2ts.tsmodel.TSComplexElement;
+import com.blueveery.springrest2ts.tsmodel.TSField;
+import com.blueveery.springrest2ts.tsmodel.TSInterface;
+import com.blueveery.springrest2ts.tsmodel.TSModule;
+import com.blueveery.springrest2ts.tsmodel.TSType;
+import com.blueveery.springrest2ts.tsmodel.generics.IParameterizedWithFormalTypes;
+import com.blueveery.springrest2ts.tsmodel.generics.TSClassReference;
+import com.blueveery.springrest2ts.tsmodel.generics.TSInterfaceReference;
+import com.blueveery.springrest2ts.tsmodel.generics.TSParameterizedTypeReference;
+
+import java.lang.reflect.AnnotatedType;
+import java.util.List;
+import java.util.SortedSet;
+
+public class ModelClassesToTsClassesConverter extends ModelClassesAbstractConverter {
+
+    public ModelClassesToTsClassesConverter(
+            ImplementationGenerator implementationGenerator, ObjectMapper objectMapper
+    ) {
+        super(implementationGenerator, objectMapper);
+    }
+
+    public ModelClassesToTsClassesConverter(
+            ImplementationGenerator implementationGenerator, ClassNameMapper classNameMapper, ObjectMapper objectMapper
+    ) {
+        super(implementationGenerator, classNameMapper, objectMapper);
+    }
+
+    @Override
+    public boolean preConverted(JavaPackageToTsModuleConverter javaPackageToTsModuleConverter, Class javaClass) {
+        if (TypeMapper.map(javaClass) == TypeMapper.tsAny) {
+            ObjectMapper objectMapper = selectObjectMapper(javaClass);
+            if (objectMapper.filterClass(javaClass)) {
```

Comment:
It can be simplified to:
```
 if (objectMapper.filterClass(javaClass)) {
                TSModule tsModule = javaPackageToTsModuleConverter.getTsModule(javaClass);
                TSType tsType = (javaClass.isInterface()) ?
                        new TSInterface(createTsClassName(javaClass), tsModule)
                        : new TSClass(createTsClassName(javaClass), tsModule, getImplementationGenerator());
                tsModule.addScopedElement((TSScopedElement) tsType);
                TypeMapper.registerTsType(javaClass, tsType);
                return true;
            }
```

Reply:
`if(TypeMapper.map(javaClass) == TypeMapper.tsAny)` this condition is required cos it could come from mapping set in TypeMapper not from class filtering if I correctly understand You